src/runtime/module.cc

```diff
@@ -97,7 +97,7 @@ Module Module::LoadFromFile(const std::string& file_name,
 }
 
 bool Module::IsEmpty() const {
-  return node_.get() == nullptr;
+  return this->operator->() == nullptr;
```

Comment:
does apache/tvm have this change?

Reply:
I don't think Apache TVM has this. Can you just use `return this->defined();` as `Module` is an `ObjectRef`?

Reply:
> does apache/tvm have this change?

No, apache/tvm does not have this. This is used for our TRT implementation.


> I don't think Apache TVM has this. Can you just use `return this->defined();` as `Module` is an `ObjectRef`?

Yes, I will try that.